packages/next/server/google-font-metrics.ts

```diff
@@ -0,0 +1,11090 @@
+export const googleFontsMetrics = {
```

Comment:
Should this be a JSON file so that we don't have to process it with TypeScript?

Also do we need to worry about these metrics getting out of date if any of the fonts are updated?

Reply:
Updated in https://github.com/vercel/next.js/pull/40185/commits/8cb7714cab4e189eb49fdc30439d66a9d49cc206